src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AppointmentService.kt

```diff
@@ -70,25 +73,43 @@ class AppointmentService(
     )
   }
 
+  fun createAppointment(createAppointmentSpecification: CreateAppointmentSpecification): AppointmentCreatedDto {
+    val event = prisonApiService.postAppointment(
+      createAppointmentSpecification.bookingId!!,
```

Comment:
the `!!` here looks wrong, dto perhaps should not accept nulls?

Reply:
You're right. It shouldn't